src/main/java/dev/hbeck/kdl/parse/KDLParser.java

```diff
@@ -485,22 +488,24 @@ String parseEscapedString(KDLParseContext context) throws IOException {
         }
     }
 
-    int getEscaped(int c, KDLParseContext context) throws IOException {
+    String getEscaped(int c, KDLParseContext context) throws IOException {
```

Comment:
I don't think I understand the reason for the change to String here.

Reply:
Yeah on second thought there really isn't a reason to have changed to String after all 🤦‍♀️ 

I've changed it back to int